src/main/eo/org/eolang/strings/compare.eo

```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-+package org.eolang.txt
++package org.eolang.strings
```

Comment:
I changed the name of this package due to name conflict with `org.eolang.txt` from `eo-runtime` EO